test/eventJoinedPeople.test.ts

```diff
@@ -8,11 +8,12 @@ import { Message } from '../src/config/messages'
 
 beforeAll(() => build())
 afterAll(() => sequelize.close())
-
+const token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwidXNlcm5hbWUiOiJtb3N0YWZhIiwiaWF0IjoxNjY2NTExODQwfQ.LUdcIEgpLB_4m9p8tnQGVKwihZyqlEob_7HsQSXZUgc'
```

Comment:
i thinks this token exists in our config
```suggestion
const token = config.token
```